src/main/java/org/pih/petl/Application.java

```diff
@@ -27,6 +28,10 @@ public class Application {
 
     private static final Log log = LogFactory.getLog(Application.class);
 
+    static {
+        TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
+    }
+
```

Comment:
Ultimately, I think this fix here is the main thing that was needed.  @mogoodrich unless you can think of a reason this would not be correct.  Both MySQL and SQL Server will use UTC by default I believe.  Or we can configure them to do so.  This ensures that even if the system running PETL is not UTC (eg. like my laptop) that UTC is still used by the JVM.  This eliminates tricky, hidden conversion issues happening in JDBC drivers and such.